src/pages/[[...markdownPath]].js

```diff
@@ -89,12 +89,13 @@ function reviveNodeOnClient(key, val) {
       type = Fragment;
     }
     return {
-      $$typeof: Symbol.for('react.element'),
+      $$typeof: Symbol.for('react.transitional.element'),
       type: type,
       key: key,
       ref: null,
       props: props,
       _owner: null,
+      _store: null,
```

Comment:
https://github.com/facebook/react/blob/c56c6234328a29930487295afe61597db48f058c/packages/react/src/ReactChildren.js#L245

Im unsure what this is used for and also should I update react to soft check for _store?